### PR TITLE
Set up Nightwatch VRT testing and add test for Classic buyflow

### PR DESCRIPTION
**Pull request by fikus**
Add Nightwatch VRT plugin for visual regression tests (screenshot tests). Add an initial test case for the Classic buyflow.

b/294885021, b/294868068

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@babel/core": "7.22.9",
     "@babel/eslint-parser": "7.22.9",
     "@babel/preset-env": "7.22.9",
+    "@nightwatch/vrt": "^3.0.0",
```
Comment (ChrisAntaki) [inline package.json:37]: Could this be in the `optionalDependencies` list, where the `nightwatch` dependency resides? That would keep the dependencies lighter for the unit tests GitHub action, and a normal `npm install` command would still install it

File: test/e2e/nightwatch.conf.js [hunk 2]
```diff
@@ -57,6 +63,11 @@ module.exports = {
           'populate-client-config-classic',
         ],
       },
+      '@nightwatch/vrt': {
+        baseline_suffix: '_all_experiments_on',
```
Comment (ChrisAntaki) [inline test/e2e/nightwatch.conf.js:67]: Can this also test with experiments off?

Comment (ChrisAntaki) [reply test/e2e/nightwatch.conf.js:67]: Oh I see it does, nice

File: package.json (hunk 1 above)
Comment (fikus) [reply package.json:37]: Hm, I added it to `devDependencies` based on the installation instructions. Test framework dependencies are a commonly cited example of when to use dev dependencies. IIUC, optionalDependencies are for dependencies that can be used by app code but are not strictly required.

https://docs.npmjs.com/cli/v9/configuring-npm/package-json

Our devDependencies list includes many other test packages like chai, karma, etc. I'm not sure why nightwatch and others would be in optionalDependences.

Comment (fikus) [reply package.json]: Got it, thanks. Updated.

Comment (ChrisAntaki) [reply package.json]: Thanks! Just noticed the `check` omits the optional deps too https://github.com/subscriptions-project/swg-js/blob/65ccb7d85cd8a0080fe90808a1941de3cf083263/.github/workflows/ci.yaml#L22

**Review by ChrisAntaki — APPROVED** [review]
Thanks for adding this

**Pull request merged**